hoot-core/src/main/cpp/hoot/core/algorithms/linearreference/WaySublineRemover.cpp

```diff
@@ -44,7 +46,8 @@ namespace hoot
 {
 
 std::vector<ElementId> WaySublineRemover::removeSubline(
-  const WayPtr& way, const WaySubline& subline, OsmMapPtr& map)
+  const WayPtr& way, const WaySubline& subline, OsmMapPtr& map,
```

Comment:
This method is used by `LinearDiffMerger` (called by `DiffConflator`) to make the partial feature merges.